builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -28,511 +29,521 @@ import groovy.transform.Field
+  echo "serverless file data $serverlessyml"
 }
 
 def LoadConfiguration() {
-	def result = readFile('deployment-env.yml').trim()
-	echo "result of yaml parsing....$result"
-	def prop = [:]
-	def resultList = result.tokenize("\n")
-
-	// delete commented lines
-	def cleanedList = []
-	for (i in resultList) {
-		if (i.toLowerCase().startsWith("#")) { } else {
-			cleanedList.add(i)
-		}
-	}
-	// echo "result of yaml parsing after clean up....$cleanedList"
-	for (item in cleanedList) {
-		// Clean up to avoid issues with more ":" in the values
-		item = item.replaceAll(" ", "").replaceFirst(":", "#");
-		def eachItemList = item.tokenize("#")
-		//handle empty values
-		def value = null;
-		if (eachItemList[1]) {
-			value = eachItemList[1].trim();
-		}
-
-		if (eachItemList[0]) {
-			prop.put(eachItemList[0].trim(), value)
-		}
-
-	}
-	echo "Loaded configurations....$prop"
-	return prop
+  def result = readFile('deployment-env.yml').trim()
+  echo "result of yaml parsing....$result"
+  def prop = [: ]
+  def resultList = result.tokenize("\n")
+
+  // delete commented lines
+  def cleanedList = []
+  for (i in resultList) {
+    if (i.toLowerCase().startsWith("#")) {} else {
+      cleanedList.add(i)
+    }
+  }
+  // echo "result of yaml parsing after clean up....$cleanedList"
+  for (item in cleanedList) {
+    // Clean up to avoid issues with more ":" in the values
+    item = item.replaceAll(" ", "").replaceFirst(":", "#");
+    def eachItemList = item.tokenize("#")
+    //handle empty values
+    def value = null;
+    if (eachItemList[1]) {
+      value = eachItemList[1].trim();
+    }
+
+    if (eachItemList[0]) {
+      prop.put(eachItemList[0].trim(), value)
+    }
+
+  }
+  echo "Loaded configurations....$prop"
+  return prop
 }
 
 
 /**
 	Build project based on runtime
 */
 def buildLambda(runtime) {
-	echo "installing dependencies for $runtime"
-	if (runtime.indexOf("nodejs") > -1) {
-		sh "npm install --save"
-	} else if (runtime.indexOf("java") > -1) {
-		sh "mvn package"
-	} else if (runtime.indexOf("python") > -1) {
-		// install requirements.txt in library folder, these python modules
-		// will be a part of deployment package
-		sh "pip install -r requirements.txt -t library"
-
-		// create virtual environment and install dev-requirements
-		sh """
-		pip install virtualenv
-		virtualenv venv
-		. venv/bin/activate
-		pip install pytest
-		"""
-	}
+  echo "installing dependencies for $runtime"
+  if (runtime.indexOf("nodejs") > -1) {
+    sh "npm install --save"
+  } else if (runtime.indexOf("java") > -1) {
+    sh "mvn package"
+  } else if (runtime.indexOf("python") > -1) {
+    // install requirements.txt in library folder, these python modules
+    // will be a part of deployment package
+    sh "pip install -r requirements.txt -t library"
+
+    // create virtual environment and install dev-requirements
+    sh ""
+    "
+    pip install virtualenv
+    virtualenv venv
+      .venv / bin / activate
```

Comment:
please restore this change.  Also please remove the plugin or extension that is doing formatting for you as this issue is being repeated everytime I see a PR from you that formats groovy file.

Reply:
Done